cronsul

```diff
@@ -39,9 +52,8 @@ esac
 
 command=("$@")
 
-# This will be wrong if cron starts our job really late.
-schedule_date="$(( ($(date '+%s') / period) * period ))"
+current_run="$(date '+%s')"
```

Comment:
With this PR, you shouldn't set your period to the same frequency as your cron runs. Instead, you should set it to the maximum time drift you want to allow between your servers. If you set a cronsul job for the top of every hour and you anticipate all machines being quite close in time, a period of 5 or 10 minutes would be perfectly sufficient to ensure that the job is run on only one member every hour.